test/kuttl/tests/glance_dist_image_import/README.md

```diff
@@ -0,0 +1,67 @@
+# Distributed Image Import kuttl test
+
+This goal of this test is to verify that it is possible to proxy the image
+upload request to the replica that has the data in its staging area. It assumes
+that `glance-direct` is enabled, and the backend used for this verification is
+not relevant. For this reason, the `glance-single` layout is deployed and we
+can avoid the extra complexity brought by the `split` layout. The openstack cli
+is used to perform actions against the glance instance resolved by keystone
+catalog, hence only external would have been affected by this test.
+
+## Kuttl test steps
+
+The steps are mostly described with an example in the
+[dist-image-import](../../../../config/samples/distributed_image_import/README.md)
+document that gives an overview of the feature.
+We assume two `GlanceAPIs` exist, and the steps perform a `per-staged` upload
```

Comment:
I don't know if there's a better term I can use, perhaps I can try to use a better wording (feel free to suggest of course), but per-staged approach in my mind is basically  a "multi stage" approach, where, instead of uploading an image with a single command, you first create an empty box, then stage your data on replica1, then upload the image using replica2 (that asks data to replica1).
Now that I'm writing this, I realize this term can be confusing, I'll change that in my next iteration (after CI runs).

Reply:
However I think it's a good idea the usage of `pre-staged`, considering that the real upload command uses pre-staged data from another replica. I like it, I'll change it as you suggested!

Reply:
Done and updated.